### PR TITLE
Deprecate SbFileGetPathInfo

### PR DESCRIPTION
**Pull request by madhurajayaraman**
b/302715109

### CODE REVIEW

File: third_party/googletest/src/googletest/include/gtest/internal/gtest-port.h [hunk 1]
```diff
@@ -2072,12 +2073,12 @@ namespace posix {
 
 #if GTEST_OS_STARBOARD
 
-using StatStruct = SbFileInfo;
+typedef struct stat StatStruct;
```
Comment (kaidokert) [inline third_party/googletest/src/googletest/include/gtest/internal/gtest-port.h:2076]: Please leave it as `using StatStruct = stat`;

**Review by kaidokert — APPROVED** [review]
Looks okay to me. Of course please make sure tests pass

**Review by y4vor — APPROVED** [review]

File: third_party/googletest/src/googletest/include/gtest/internal/gtest-port.h (hunk 1 above)
Comment (madhurajayaraman) [reply third_party/googletest/src/googletest/include/gtest/internal/gtest-port.h:2075]: https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/COBALT:third_party/googletest/src/googletest/include/gtest/internal/gtest-port.h;l=2210

We also use `typedef struct stat StatStruct` in a couple of other places in the gtest file -- is there a benefit to using `using` instead?

Comment (kaidokert) [reply third_party/googletest/src/googletest/include/gtest/internal/gtest-port.h:2075]: No other benefit than consistent style both in template code and outside of it. If the compiler barks, it's fine to leave as is.

**Pull request merged**